infrastructure/modules/aws/ecr/versions.tf

```diff
@@ -2,7 +2,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 4.18.0"
+      version = ">= 4.47"
```

Comment:
Is this the minimal version where force_delete is supported? or just the latest version?

Reply:
I took 4.47 version because of aws eks module v19 : https://github.com/terraform-aws-modules/terraform-aws-eks/blob/v19.10.0/versions.tf#L7